build.gradle

```diff
@@ -95,30 +95,30 @@ dependencies {
     implementation group: 'com.truelayer', name: 'truelayer-signing', version: '0.2.4'
 
     // Serialization
-    def jacksonVersion = '2.13.4'
```

Comment:
We need to be aware that Dependabot is not smart enough to catch updates if dependencies are declared like this :(